plugins/design-system/components/component-example/ComponentExampleOverrides.js

```diff
@@ -0,0 +1,7 @@
+const ComponenxtExampleOverrides = {};
+
+["ReactCode", "HtmlCode", "Preview"].forEach(function(element) {
```

Comment:
I haven't dived too deeply into it yet. My initial impression is that it is more than what I need for my use case.